packages/subgraph/src/mappings/superfluidGovernance.ts

```diff
@@ -61,6 +89,31 @@ export function handleCFAv1LiquidationPeriodChanged(
     ev.isKeySet = event.params.isKeySet;
     ev.liquidationPeriod = event.params.liquidationPeriod;
     ev.save();
+
+    // Update cfav1 liquidation period for token or default governance configs
+    if (event.params.superToken.equals(ZERO_ADDRESS)) {
+        let defaultGovernanceConfigs = DefaultGovernanceConfigs.load(
```

Comment:
Why not use `getOrInitDefaultGovernanceConfigs` here?

Reply:
oh derp, good point, the pushed fix resolves this comment and the one above regarding the updatedAt fields...